Add Distributed Blob Publishing to Fulu networking spec (#4183)
ethereum/consensus-specs: specs/fulu/validator.md

The sequence of sidecars associated with a block and can be obtained by first computing
`cells_and_kzg_proofs = [compute_cells_and_kzg_proofs(blob) for blob in blobs]` and then calling
`get_data_column_sidecars_from_block(signed_block, cells_and_kzg_proofs)`.

Moreover, the full sequence of sidecars can also be computed from `cells_and_kzg_proofs` and any single

> `data_column_sidecar`, by calling `get_data_column_sidecars_from_column_sidecar(sidecar, cells_and_kzg_proofs)`.

Reviewer: One more nit, the variable "data_column_sidecar" at the beginning of the sentence doesn't match the first argument "sidecar" to the function. I think these should be the same.

Author: Updated the `data_column_sidecar` here to just `sidecar to match the function, and removed the blank line below. Thanks!

The sequence of sidecars associated with a block and can be obtained by first computing
`cells_and_kzg_proofs = [compute_cells_and_kzg_proofs(blob) for blob in blobs]` and then calling
`get_data_column_sidecars_from_block(signed_block, cells_and_kzg_proofs)`.

Moreover, the full sequence of sidecars can also be computed from `cells_and_kzg_proofs` and any single
`sidecar`, by calling `get_data_column_sidecars_from_column_sidecar(sidecar, cells_and_kzg_proofs)`.
This can be used in distributed blob publishing, to reconstruct all sidecars from any sidecar received

> on the wire, as well as all cells and kzg proofs retrieved from the local execution layer client.

Reviewer 2: minor, but i think this language is a bit more clarifying
```suggestion
on the wire, assuming all cells and kzg proofs could be retrieved from the local execution layer client.
```
